index.html

```diff
@@ -1049,6 +1049,371 @@ <h2>Evidence</h2>
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+
+   
+   <section>
+      <h2>Holder Acts on Behalf of Subject</h2>
+
+      <p>
+When the holder wants to act on behalf of the subject, the holder creates 
+a Profile, in which the profile ID is the ID of the holder, and the relationship 
+property states the relationship between the holder and the subject.
+
+     </p>
+     <p>
+If the holder wants to act on behalf of two or more different subjects, then
+each relationship property states the holder's relationship to one the subjects.
+      </p>
+
+     <p>
+Note. The verifier still has to validate that these relationships are true.
+     </p>
+
+   <section>
+      <h3>The Relationships Property</h3>
+
+      <p>
+The Relationships property states the Profile holder's relationship with each of the 
+subjects of the embedded credentials. Each value comprises the relationship property 
+(i.e. a string describing the relationship of the holder to the subject) and 
+the ID of the subject being referred to.
+
+      </p>
+ 
+      <p>
+Note that values of the relationship string will be defined in other standards, but it is 
+likely that typical values might be "parent" (when the subject is a child) and 
+"owner" (when the subject is a pet) etc.
+      </p>
+     
+        <pre class="example nohighlight" title="An example of the relationships 
+property, where a parent asserts that their child is under 16">
+
+{
+	"id": "did:example:e12ec3ebf12eb1c276cv211f712",
+	"type": ["VerifiableProfile"],
+	"holderid": "did:example:ebfeb1c276e12ec211f712ebc6f",
+	"relationships": [{
+		"subjectid": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"relationship": "parent"
+	}],
+	"verifiableCredential": [{
+		"id": "http://dmv.example.gov/credentials/3732",
+		"type": ["Credential", "ProofOfAgeCredential"],
+		"issuer": "https://dmv.example.gov/issuers/14",
+		"issued": "2010-01-01T19:73:24Z",
+		"claim": {
+			"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+			"ageUnder": 16
+		},
+		"proof": {
+			"type": "RsaSignature2018",
+			"created": "2017-06-17T10:03:48Z",
+			"creator": "https://dmv.example.gov/issuers/14/keys/234",
+			"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+			"signatureValue": "pY9...Cky6Ed = "
+		}
+	}],
+	"proof": [{
+		"type": "RsaSignature2018",
+		"created": "2017-06-18T21:19:10Z",
+		"creator": "did:example:ebfeb1c276e12ec211f712ebc6f/keys/2",
+		"nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+		"signatureValue": "BavEl...W3JT24 = "
+	}]
+}
+        </pre>
+
+
+      </section>
+    </section>
+
+    <section>
+       <h2>Subject Delegates to Holder</h2>
+
+     <p>
+This is supported in the data model in two different ways: through the use of either recursion or profiles. Both methods involve the subject issuing a new credential to the delegated holder.
+    </p>
+
+    <p>
+This process of recursion could continue without restriction. Therefore 
+a new property “delegatable”, is added to the data model.
```

Comment:
It is critical to note that a delegateable flag does not absolutely prevent delegation, since [an entity can always conspire to delegate out of band](http://erights.org/elib/capability/delegations.html) (This is not a VC ACL vs ocap thing, this is just a general property about delegation and its inability to be completely prohibited.)

Reply:
@cwebber Sure, no-one cannot prevent anyone from delegating anything to anyone else. Just like anyone can become a CA and issue an X.509 cert to anyone by signing the appropriate ASN.1 data structure. But you are missing the point, which is TRUST. The verifier knows who it trusts, and it will not trust me to be a CA or you to delegate a "president of the USA" VC to me.
So the point is, if a verifier's TRUSTED issuer says 'this VC is not to be delegated" then the standard conforming verifier's software will not trust any VC that is delegated from it. Thus in my opinion this is a non-issue, and you are wrong. The terms of use field can be mandatory and can be enforced by the standard conforming software of the verifier.

On a related point, any verifier (and its software) is allowed to be non-conformant to the standard. In other words, a verifier's software is allowed to ignore all the rules in the standard about how to process the various fields, and to do what it wants to do (since the verifier bears all the risk). It could accept that I am a CA, or the president of the USA and let me access all its treasured resources. Well, if you will run non-standard software, what do you expect?